src/numeric_filter.c

```diff
@@ -66,12 +73,12 @@ NumericFilter *NumericFilter_Parse(ArgsCursor *ac, QueryError *status) {
 
   // Parse the min range
   const char *s = AC_GetStringNC(ac, NULL);
-  if (parseDoubleRange(s, &nf->inclusiveMin, &nf->min, 1, status) != REDISMODULE_OK) {
+  if (parseDoubleRange(s, &nf->inclusiveMin, &nf->min, 1, 1, status) != REDISMODULE_OK) {
```

Comment:
Since legacy `FILTER` does not support parameters and sign is implicitly `+` ?

Reply:
Yes. And a test was added to show that parameters are not supported.